src/iperf_api.c

```diff
@@ -1881,17 +1883,73 @@ iperf_check_throttle(struct iperf_stream *sp, struct iperf_time *nowP)
     struct iperf_time temp_time;
     double seconds;
     uint64_t bits_per_second;
+    int64_t missing_rate;
+    uint64_t bits_sent;
+    
+#if defined(HAVE_CLOCK_NANOSLEEP) || defined(HAVE_NANOSLEEP)
+    struct timespec nanosleep_time;
+    int64_t time_to_green_ligh, delta_bits;
```

Comment:
`time_to_green_ligh` should probably be `time_to_green_light`

Reply:
Sure ... Changed the `time_to_green_light`.